WireAPI/Sources/WireAPI/Private/APIs/BackendInfoAPI/BackendInfoAPIV0.swift

```diff
@@ -0,0 +1,76 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+class BackendInfoAPIV0: BackendInfoAPI {
```

Comment:
For me it's confusing we have this separation of folder `Private` and `Public`, because it forces me to scroll through the package and jump in navigation.

What do you think to organise the folders by functionality like `Core`, `APIs/BackendInfo`?

Reply:
My aim was to try make it as clear as possible what constitutes the public interface and what is an implementation detail. But I agree that there's a bit of jumping back and forth. I'm happy to reorganize by functionality but we should still be vigilant to maintain correct access control.

Reply:
done